donatefood/home/templates/home/home.html

```diff
@@ -48,12 +48,15 @@ <h1 class="w3-margin w3-jumbo">Food Bank Finder</h1>
 
       <div class="w3-twothird">
         <h1>Donate to Your Local Food Bank</h1>
-        <h5 class="w3-padding-32">Insert sentimentally-convincing statement here</h5>
+        <h5 class="w3-padding-32">Giving is not just about making a donation, it's about making a difference</h5>
         <div class="w3-twothird">
-          <h1>Lorem Ipsum</h1>
-          <h5 class="w3-padding-32">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-            incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
-            laboris nisi ut aliquip ex ea commodo consequat.</h5>
+          <h1>Why Donate?</h1>
+          <h5 class="w3-padding-32">
+Donating to charity is a major mood-booster. The knowledge that you’re helping others is hugely empowering and, in turn, can make you feel happier and more fulfilled. Research has identified a link between making a donation to charity and increased activity in the area of the brain that registers pleasure - proving that as the old adage goes, it really is far better to give than to receive.
+
+Our own research into why people give supports this. We asked 700 of our generous donors to tell us what motivates them to give regularly to charity; 42% agreed the enjoyment they receive from giving as a key influence.  
```

Comment:
We have not done this and should not advertise falsely, especially if we release this on the internet.